domain/modelconfig/service/service.go

```diff
@@ -314,12 +315,34 @@ func (v *spaceValidator) HasSpace(ctx context.Context, spaceName string) (bool,
 	return v.st.SpaceExists(ctx, spaceName)
 }
 
+// setModelConfigValidator returns a config validator to use on model config
+// when it is being set initially. The validator returned will check that:
+// - Network space exists.
+// - Secret backend exists.
+// - Container networking method is valid.
+func (s *Service) setModelConfigValidator(
```

Comment:
`setModelConfigValidator` does not set anything. Maybe just `modelConfigValidator`.

Reply:
I've renamed these `validatorForSetModelConfig` and `validatorForUpdateModelConfig`, hope that's more clear.